polling_stations/apps/councils/management/commands/import_councils.py

```diff
@@ -18,6 +15,21 @@ def union_areas(a1, a2):
     return MultiPolygon(a1.union(a2))
 
 
+NIR_IDS = [
+    "ABC",
+    "AND",
+    "ANN",
+    "BFS",
+    "CCG",
+    "DRS",
+    "FMO",
+    "LBC",
+    "MEA",
+    "MUL",
+    "NMD",
```

Comment:
If the councils will have register codes attached to them in the DB, could you infer this based on the register codes on councils where `council_id__startswith="N09"` instead of hard-coding it?

Reply:
This would be a good idea for the longer term, but this is a temporary work around: At the moment the name field is NULLABLE, and in the live API all the NI names are NULL. I think this is because the display name != council name, but I'd rather the EC have control over the display name in all cases, so we're working on making the name field required. 

When / if it is, or when they have populated the name field in all cases, this code won't be needed.

Because of that, I'll just leave it for the moment!